node-react/packages/backend/package.json

```diff
@@ -2,27 +2,23 @@
   "name": "backend",
   "version": "0.1.0",
   "private": true,
+  "type": "module",
   "scripts": {
-    "start": "NODE_ENV=development nodemon src/index.js --exec babel-node"
+    "start": "NODE_ENV=development nodemon src/index.js"
   },
   "dependencies": {
     "cors": "^2.8.5",
-    "express": "^4.17.3"
+    "express": "^4.18.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.17.5",
-    "@babel/eslint-parser": "^7.17.0",
-    "@babel/node": "^7.16.8",
-    "@babel/plugin-proposal-class-properties": "^7.16.7",
-    "@babel/plugin-proposal-object-rest-spread": "^7.17.3",
-    "@babel/preset-env": "^7.16.11",
```

Comment:
Not 100% sure why we needed babel here, everything appears to work without it. @branclar if you can think of anything LMK.